packages/libs/error-stack/Cargo.toml

```diff
@@ -1,7 +1,16 @@
 [package]
 name = "error-stack"
 version = "0.0.0"
+authors = ["HASH"]
 edition = "2021"
+rust-version = "1.61.0"
+description = "A context-aware error library with abritrary attached user data"
+documentation = "https://docs.rs/error-stack"
+readme = "README.md"
+homepage = "https://hash.ai"
```

Comment:
@nonparibus Should this point to `hash.ai` or to the repository?

Reply:
Let's link this to https://github.com/hashintel/hash/tree/main/packages/libs/error-stack for the timebeing.

When we have dedicated docs/a landing page for this lib hosted on the https://hash.dev/ site we should update the link to point there.